Non-root users are allowed to change the unicode mode of their console, but
not the global keymap.  root will have to load the keymap in unicode mode
explicitly.

--- kbd-1.12/src/unicode_start.no-user-map	2005-12-01 19:33:19.000000000 +0100
+++ kbd-1.12/src/unicode_start	2005-12-01 19:33:28.000000000 +0100
@@ -14,12 +14,6 @@
 
 kbd_mode -u
 
-# Change the keyboard mapping in such a way that the non-ASCII keys
-# produce UTF-8 encoded multibyte sequences, instead of single bytes
-# >= 0x80 in a legacy 8-bit encoding.
-
-dumpkeys | loadkeys --unicode
-
 # 2. The output side: the console screen.
 
 # Tell the console output driver that the bytes arriving are UTF-8
